src/main/routes/home.ts

```diff
@@ -3,10 +3,26 @@ import { Application } from 'express';
 import { Redis } from 'ioredis';
 
 export default async function (app: Application): Promise<void> {
-  const redis = new Redis(config.get('pcs.redis-connection-string'));
-  await redis.set('mykey', 'value');
   app.get('/', async (req, res) => {
-    const mykey = await redis.get('mykey');
-    res.render('home', { redisKey: mykey });
+    let logMsg = '';
+    if (!app.locals.redis) {
+      try {
+        app.locals.redis = new Redis(config.get('pcs.redis-connection-string'), {
+          maxRetriesPerRequest: 3,
+        });
+        await app.locals.redis.set('mykey', 'value');
+      } catch (error) {
+        logMsg = `${error.message} - Unable to connect and Set to redis`;
+        app.locals.redis.disconnect();
```

Comment:
A potential issue for future that app might still serve traffic without connection

Reply:
i will add a ticket in the backlog to serve the traffic connectivity as means of keeping the design future